### PR TITLE
docs: fix MSBuild property InvariantGlobalization

### PR DESCRIPTION
**Pull request by imba-tjd**
https://docs.microsoft.com/en-us/dotnet/core/run-time-config/globalization

This option seems to have been changed.

### CODE REVIEW

File: docs/design/features/globalization-invariant-mode.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ Applications can enable the invariant mode by either of the following:
 
     ```xml
     <ItemGroup>
-      <RuntimeHostConfigurationOption Include="System.Globalization.Invariant" Value="true" />
+        <InvariantGlobalization>true</InvariantGlobalization>
```
Comment (tarekgh) [inline docs/design/features/globalization-invariant-mode.md:107]: @eerhardt do you know if this is something changed?

Comment (eerhardt) [reply docs/design/features/globalization-invariant-mode.md:107]: The only meaningful change was that the ILLinker added a new metadata item “Trim=true” to this item in order to be respected by the linker. So the current MSBuild xml in the docs isn’t enough for the linker to work.

Using the property, which is the proposed change, is the right call. We should be telling users to set the properties.

File: docs/design/features/globalization-invariant-mode.md [hunk 2]
```diff
@@ -104,7 +104,7 @@ Applications can enable the invariant mode by either of the following:
 
     ```xml
     <ItemGroup>
```
Comment (eerhardt) [inline docs/design/features/globalization-invariant-mode.md:106]: This needs to be PropetyGroup, not ItemGroup

**Review by eerhardt — CHANGES_REQUESTED** [review]

File: docs/design/features/globalization-invariant-mode.md (hunk 2 above)
Comment (imba-tjd) [reply docs/design/features/globalization-invariant-mode.md]: Done.

**Review by eerhardt — APPROVED** [review]
Lgtm

**Pull request merged**